website/source/docs/post-processors/checksum.html.md

```diff
@@ -0,0 +1,43 @@
+---
+description: |
+    The checksum post-processor computes specified checksum for the artifact list
+    from an upstream builder or post-processor. All downstream post-processors will
+    see the new artifacts. The primary use-case is compute checksum for artifacts
+    allows to verify it later.
+layout: docs
+page_title: 'Checksum Post-Processor'
+...
+
+# Checksum Post-Processor
+
+Type: `checksum`
+
+The checksum post-processor computes specified checksum for the artifact list
+from an upstream builder or post-processor. All downstream post-processors will
+see the new artifacts. The primary use-case is compute checksum for artifact to
+verify it later.
```

Comment:
I think we should also include a note about how the checksum file will be written, so it is more clear what this post-processor is going to do for you. E.g:

1. Create a checksum file
2. Pass along the original artifact for additional post-processors